utils/roadiz/entity.ts

```diff
@@ -28,20 +29,7 @@ export function isNodeType(entity: unknown): entity is JsonLdObject {
     return !!(entity && typeof entity === 'object' && '@id' in entity && '@type' in entity)
 }
 
-// NODE SOURCE
-export function isPageEntity(entity: JsonLdObject): boolean {
-    return isEntityType(entity, 'Page')
-}
-
-export function isBlogPostEntity(entity: JsonLdObject): boolean {
-    return isEntityType(entity, 'BlogPost')
-}
-
-export function isBlogListingEntity(entity: JsonLdObject): boolean {
-    return isEntityType(entity, 'BlogPostContainer')
-}
-
-// BLOCKS
-export function isContentBlock(entity: JsonLdObject): boolean {
-    return isEntityType(entity, 'ContentBlock')
+// ROADIZ NODE SOURCE
+export function isPageEntity(entity: JsonLdObject | undefined): entity is NSPage {
```

Comment:
IMO this is not the place for testing if `entity` is `undefined`. The function has only one goal, ie testing the entity type